pkg/rules/rules.go

```diff
@@ -0,0 +1,139 @@
+package rules
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/oklog/run"
+	"github.com/pkg/errors"
+
+	"github.com/prometheus/client_golang/prometheus"
+	prometheus_config "github.com/prometheus/prometheus/config"
+	"github.com/prometheus/prometheus/discovery"
+	"github.com/prometheus/prometheus/model/labels"
+	"github.com/prometheus/prometheus/notifier"
+	prom_rules "github.com/prometheus/prometheus/rules"
+
+	"github.com/timescale/promscale/pkg/log"
+	"github.com/timescale/promscale/pkg/pgclient"
+	"github.com/timescale/promscale/pkg/query"
+	"github.com/timescale/promscale/pkg/rules/adapters"
+)
+
+const (
+	DefaultQueueCapacity   = 10000
```

Comment:
do these need to be configurable?

Reply:
Prometheus allows it, so we should.

Reply:
Should we create an issue for this then? Or is this for an upcoming PR?

Reply:
Upcoming PR